pkg/mco/mcplist.go

```diff
@@ -12,10 +12,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-func ListMCP(apiClient *clients.Settings, listOptions metav1.ListOptions) ([]*MCPBuilder, error) {
-	glog.V(100).Infof("Listing all MCP resources with the options %v", listOptions)
+func ListMCP(apiClient *clients.Settings, listOptions ...metav1.ListOptions) ([]*MCPBuilder, error) {
+	passedOptions := metav1.ListOptions{}
 
-	mcpList, err := apiClient.MachineConfigPools().List(context.Background(), listOptions)
+	if len(listOptions) == 1 {
+		passedOptions = listOptions[0]
+	} else if len(listOptions) > 1 {
+
+		return nil, fmt.Errorf("error: more than one ListOptions was passed")
+	}
+
+	glog.V(100).Infof("Listing all MCP resources with the options %v", passedOptions)
```

Comment:
Do we want to show this if the passedOptions is empty ?

Reply:
No, it assumes the variable is populated. I'll condition the glog messages accordingly